Ryujinx.HLE/HOS/Applets/AppletManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Ryujinx.HLE.HOS.Applets
+{
+    static class AppletManager
+    {
+        private static Dictionary<AppletId, Type> _appletMapping;
```

Comment:
```suggestion
        private static Dictionary<AppletId, Type> _appletList;
```

Maybe `List` is more revelant than `Mapping`.

Reply:
Well, it maps AppletId to Applet implementation?

Reply:
I think mapping makes more sense, since that is not exactly a list.

Reply:
Then it's fine.